.gitignore

```diff
@@ -62,7 +62,7 @@ buildNumber.properties
 
 # MOSAIC
 
-tmp/
```

Comment:
why is this removed?

Reply:
It was previously mentioned twice, see line 7